lms/djangoapps/course_api/views.py

```diff
@@ -121,15 +125,19 @@ class CourseListView(ListAPIView):
 
     **Parameters**
 
-        username (optional):
+        username:
             The username of the specified user whose visible courses we
-            want to see.  Defaults to the current user.
+            want to see.
 
         org (optional):
             If specified, visible `CourseOverview` objects are filtered
             such that only those belonging to the organization with the provided
             org code (e.g., "HarvardX") are returned. Case-insensitive.
 
+        mobile (boolean):
```

Comment:
Good point.  I didn't include it since there's no need for it at the moment by our clients - and mobile_available is meant to be a short-term flag on our courses.  Hopefully, in 6 months time all of our courses will be mobile_available.  So for now, it's just a filter.